src/environment_setup/python_package.ml

```diff
@@ -72,5 +72,5 @@ let default ~host ~run_program ~install_path () =
     create_python_tool ~host ~run_program ~install_path
       ~version:"0.0.21" (Pip, Package_PyPI "topiary");
     create_python_tool ~host ~run_program ~install_path
-      ~version:"0.3.1" (Pip, Package_PyPI "vaxrank");
+      ~version:"0.4.0" (Pip, Package_PyPI "vaxrank");
```

Comment:
I don't think so; isn't that for use with FFPE/RiboZero prepped samples? /cc @iskandr